js/libs/form.js

```diff
@@ -114,6 +114,56 @@ Fliplet().then(function() {
       };
     }
 
+    function loadEntryForStorage(field) {
```

Comment:
This function only has a few lines different from `loadEntryForQuery`.

I suggest using a single function.

```js
function loadFieldValueFromSource(field, source) {
  // Run a switch-case on source
  // Use different method to get the result based on the value of source
  // Check for required parameter if necessary, e.g. field.defaultValueKey
  // Run the same code as you have now to apply the result to the field
}
```

Reply:
done